### DISCUSSION TITLE
Welcome to esp32Micronet Discussions!

### DISCUSSION THREAD

**Discussion by mnsnoop** [mnsnoop/esp32Micronet#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by circus37** [comment]
hello great job i am trying to compile the code but my arduino ide gives me a bad card error i don't know if this is the code what setting do you use for an esp32 wroom thank you very much i will cross the atlantic at the end of the month i  would tell you if it worked permanently have a nice day

**Comment by mnsnoop** [comment]
I believed I used the "ESP32 Dev Module" board but can't check right now. AFAIK it should compile under any esp32 board profile. Can you paste your build log here?

**Comment by circus37** [comment]
Ok i m sailong now but i make thos thanks

**Reply by circus37** [reply]
Sorry i look my build log hihi one ligne is not visible in the windows i copy one librairie out off his folder now everything is good thanks a lot i receive the material in one week and big test throught the atlantic for the n2k bridge if it work well my pilot have wind thanks a lot

**Comment by mnsnoop** [comment]
Good to hear, enjoy your sail.

**Reply by circus37** [reply]
good evening I have a problem with my wind transducer I have a race master as a screen and just an anemometer I can't manage to pair them I succeeded twice when I replaced the battery then nothing unfortunately I couldn't solder the  cc1001 my soldering iron is too big being in cape verde i can't order another cc1001 is there a way to check that the t120 is emitting?  if you have a test technique to confirm a failure I have an rtl sdr

**Comment by mnsnoop** [comment]
When you first turn on a display after a short 2 to 4 second period it will transmit continuously for 4 to 6 seconds. That's the easiest way to catch it transmitting. If you put the display close enough to the sdr antenna and turn the gain up you should be able to see and hear it transmit every second. These transmissions are short but should show up.

**Comment by circus37** [comment]
hello this is for the screen do you know if the anemometer emits
permanently or if it goes to sleep?  I saw the signals pass during the
pairing but when I listen to my anemometer I still don't see any signals
thank you for taking the time to answer me I tried to change the frequency
but still nothing I will charge it fully in the sun but  the pole eats at
2.9 v I think that's enough?  have a good day

**Comment by mnsnoop** [comment]
It goes to sleep. The anemometer will only transmit when asked by another device. 2.9v is enough for it to function.

**Comment by circus37** [comment]
Ok thanks i think it is sleep and the race master don t wake up it or is
dead 😖

**Comment by mnsnoop** [comment]
The 4 to 6 second continuous transmission when a display is turned on is the wake up signal. If the display is sending that signal out then the issue is with the anemometer or pairing of the two.

**Comment by circus37** [comment]
Ok thanks maybe is paired whith another device but i don t know how to
reset the anemometer thanks a lot now sailing throught the atlantic i see
that later no wind sensor all right
***@***.***/ my boat homemade building

**Comment by mnsnoop** [comment]
There is no reset. It will pair with any display that is close enough. Happy sailing!
